tools/clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -2987,7 +2986,6 @@ bool operator<(const BlockLayoutChunk &l, const BlockLayoutChunk &r) {
 }
 }
 
-#if 0 // HLSL Change - no block support
```

Comment:
Since you are removing this, you should also remove the "#endif" at line 3133?

Reply:
The corresponding endif removal is on line 1600 because It's not a removal of an ifdefed out region, it's an expansion of two that happens to merge them.